docs/integrations/databases/Druid.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: "Druid"
+---
+
+![](https://druid.apache.org/img/druid_nav.png)
+
+## Add credentials
+
+1. Create a new pipeline or open an existing pipeline.
+2. Expand the left side of your screen to view the file browser.
+3. Scroll down and click on a file named `io_config.yaml`.
+4. Enter the following keys and values under the key named `default` (you can
+   have multiple profiles, add it under whichever is relevant to you)
+
+```yaml
+version: 0.1.1
+default:
+  DRUID_HOST: ...
+  DRUID_PASSWORD: ...
+  DRUID_PATH: /druid/v2/sql/
+  DRUID_PORT: 8082
+  DRUID_SCHEME: http
+  DRUID_USER: ...
+```
+
+<br />
+
+## Using SQL block
```

Comment:
Did you test SQL block? I think SQL block actually creates a temporary table and writes data to it. But I saw that "write" is not supported for Druid in this PR.
Maybe "Use raw SQL" could work for Druid SQL block. Just wondering whether you tested it.

Reply:
Yes I tested it it works using the SQL Raw

Reply:
ok, could you add a note in the doc that it only works for raw SQL?